src/DataLoader.js

```diff
@@ -24,17 +24,46 @@ var groupEvidencesByCode = function(features) {
     return features;
 };
 
+var setVariantData = function (authority, d) {
```

Comment:
Context: We get the variants from the source, multiple of them can be in the same position but have different mutations, so we build the mutationArray. That mutation array has some basic information plus an array of variants.
Each variant in that array has some common information such as begin, end, wildType, etc. When it comes to an external variant, we move that common information to the variant level and keep any additional external information such as predictions or consequences at an internal level. 

With this code, an external variant looks like:
```
{
  "begin":184, 
  "end":184, 
  "wildType":"F",
  "alternativeSequence":"S",
  "externalData":{
    "myLab":{
      "polyphenPrediction":"unknown",
      "polyphenScore":0,
      "siftPrediction":"deleterious",
      "siftScore":0.04
    }
  },
  "internalId":"var_F184S"
}
```
For an external variant, we have in any case to get the common data and put it in the common area. Without the deletions, we would have information duplicated and it would look like:
```
{
  "begin":184, 
  "end":184, 
  "wildType":"F",
  "alternativeSequence":"S",
  "externalData":{
    "myLab":{
      "begin":184, 
      "end":184, 
      "wildType":"F",
      "alternativeSequence":"S",
      "polyphenPrediction":"unknown",
      "polyphenScore":0,
      "siftPrediction":"deleterious",
      "siftScore":0.04
    }
  },
  "internalId":"var_F184S"
}
```